### DISCUSSION TITLE
custom Adaptive Card use - which host for custom results?

### DISCUSSION THREAD

**Discussion by marcoschmucker** [microsoft-search/pnp-modern-search#3814] [Q&A]
When trying to create a custom adaptive card in the designer (https://adaptivecards.io/designer/), we cannot find the appropriate host platform to choose in order to have a valid custom adaptive card in a search result.

We tried different versions up to 4.11.

**Version used**
4.11

**Describe the bug**
custom adaptive card cannot be rendered as card.

**Desktop (please complete the following information):**
 - Edge or Chrome

![image](https://github.com/microsoft-search/pnp-modern-search/assets/27192929/14dae90d-bbbc-45e9-bd20-0245aefc7d8a)

**Comment by wobba** [comment]
Try out https://searchlayoutdesigner.azurewebsites.net/ which is what the oob search experience uses.

**Reply by marcoschmucker** [reply]
Dear @wobba - thank you. But this layout is no adaptive card. So this means, that adaptive cards are not supported?

**Reply by wobba** [reply]
https://searchlayoutdesigner.azurewebsites.net/ does indeed create adaptive cards. Pick a sample layout to get started with, click "get started", click "Edit layout" and click the "Layout payload editor". This layout may be used in the web parts. Just ensure the managed property tokens are added correctly.
 
 Look at the sample card in the web part for reference.
